modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -14,6 +14,7 @@ You can check the V2V Conversion Pod log for errors.
 
 . Obtain the V2V Conversion Pod name by running the following command:
 +
+[source,terminal]
 ----
 $ oc get pods -n <project> | grep v2v <1>
 kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
```

Comment:
This is actually a command and example output (the `kubevirt-v2v-conversion....` is the output), so it needs to be broken up and given the `.Example output` header.

Reply:
Ah, thanks for catching that.